config/stored_requests.go

```diff
@@ -17,6 +17,10 @@ type StoredRequests struct {
 	Files bool `mapstructure:"filesystem"`
 	//If data should be loaded from file system, path should be specified in configuration
 	Path string `mapstructure:"directorypath"`
+	// Lazy-load the data.
+	//   false (default): preload all the files in cache
+	//   true: load data only when incoming requests ask for the stored resource
+	LazyLoad bool `mapstructure:"lazy_load"`
```

Comment:
Since this only applies to the files source, please consider renaming to `FilesLazyLoad` with a json name of `filesystem_ lazy_load`. Ideally, we would package the `File`, `Path`, and `LazyLoad` setting to their own group like `Postgres` and `HTTP`, but that would break existing configs. At least this approach was taken in `StoredRequestsSlim`.

Reply:
For the naming I had followed `directorypath` which is also unqualified and is converted to `Files.path`.

Will rename as you suggest, it is more clear.

On this topic, I've opened #1453 which attempts to consolidate the Slim and non-Slim versions of the configuration and provides a migration path so all the stored data configurations can be finally aligned.